src/components/color-picker/color-picker.scss

```diff
@@ -49,25 +49,26 @@ $gap--large: 12px;
   @apply text-n1h;
 
   .container {
-    width: 464px;
+    inline-size: 464px;
   }
 
   .color-field-and-slider {
-    margin-bottom: -20px;
+    margin-block-end: -20px;
   }
 
   .section {
-    padding: 0 $section-padding--large $section-padding--large;
+    padding-block: 0 $section-padding--large;
+    padding-inline: $section-padding--large;
```

Comment:
This is nice.  `padding` with 3 values always confused me before, but now this reads much clearer in code and is more i18n friendly.